core/src/main/java/hudson/model/Job.java

```diff
@@ -1210,7 +1210,8 @@ public synchronized void doConfigSubmit(StaplerRequest req,
 
             String newName = req.getParameter("name");
             final ProjectNamingStrategy namingStrategy = Jenkins.getInstance().getProjectNamingStrategy();
-            if (newName != null && !newName.equals(name)) {
+            if (newName != null && !newName.trim().equals(name.trim())) {
```

Comment:
> the initial comment didn't contain any feedback regarding the code, hence @kzantow 's comment was totally valid

@oleg-nenashev i following logical changes in this PR and said what i think, if it not enough for you, then just ask me without pointing me that i should expect from somebody full understanding.  

> your offensiveness is not constructive here

I treat as offensiveness comments in my side. If many trim operations in two lines of code is ok for you and other people likes it, then i would not mind. But i don't see any feedback to this change or PR from this commenter. 
:+1: and :-1: exists for getting quorum to get better quality.